lib/elixir/lib/module.ex

```diff
@@ -252,6 +252,9 @@ defmodule Module do
   in case any of the external resources change, see for example:
   [`mix compile.elixir`](https://hexdocs.pm/mix/Mix.Tasks.Compile.Elixir.html).
 
+  Even if the given external file does not exist, the module still
+  adds a dependency on it.
```

Comment:
The opposite is not currently true, if we remove the file, the module is not recompiled. Is it worth changing?

Reply:
Maybe we should fix it.